catalog-info.yaml

```diff
@@ -0,0 +1,65 @@
+# Declare a Backstage Component that represents your application.
+---
+# yaml-language-server: $schema=https://json.schemastore.org/catalog-info.json
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: package-spec
+  description: EPR package specifications
+  links:
+    - title: Developer Documentation
+      icon: file-doc
+      url: https://www.elastic.co/guide/en/integrations-developer/current/package-spec.html
+
+spec:
+  type: tool
```

Comment:
@alexsapran @jsoriano What should be used here ? library? spec?
https://backstage.io/docs/features/software-catalog/descriptor-format/#kind-component